pkg/cloudprovider/aws/validation.go

```diff
@@ -25,11 +25,11 @@ import (
 )
 
 // Validate cloud provider specific components of the cluster spec
-func (c *Capacity) Validate(ctx context.Context) (errs *apis.FieldError) {
+func (a *API) Validate(ctx context.Context, spec *v1alpha1.ProvisionerSpec) (errs *apis.FieldError) {
```

Comment:
Can we move these functions aws/cloudprovider.go (name change recommended above). It's only a few lines, and I think the scope is easier to encapsulate now. It feels a bit awkward that this is standing alone now that the cloudprovider API surface is so much smaller (Create, Delete, Validate, GetInstanceTypes)